tests/Library/Plugin/test-update-data.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * BoldGrid Source Code
+ *
+ * @package Boldgrid_Plugintest
+ * @copyright BoldGrid.com
+ * @version $Id$
+ * @author BoldGrid.com <wpb@boldgrid.com>
+ */
+
+namespace Boldgrid\Library\Library\Plugin;
+
+use Boldgrid\Library\Library\Configs;
+
+function wp_remote_get( $url ) {
```

Comment:
Can we add a comment about what this method is and why it's outside of the class?

Reply:
*bump